gateway_lambda/index.ts

```diff
@@ -52,6 +52,7 @@ export function generateEvents(data: FxaPayload): SqsEvent[] {
       event: allowedEvents[event],
       timestamp: Math.round(new Date().getTime() / 1000),
       user_email: events[event].email,
+      transfer_sub: events[event].transfer_sub ?? null,
```

Comment:
Do we need to set `transfer_sub` to `null` here or can it be `undefined`. The reason I ask is because in the sqs_lambda you do check if `transfer_sub` is false-y. Just curious if we need to explicitly set it to `null` here 🤔

Reply:
no particular reason -prefer explicitly to null as the value is not provided from upstream request. i think the !variable (when null) is falsy right? https://developer.mozilla.org/en-US/docs/Glossary/Falsy